### ISSUE TITLE
First things to work on before a first prototype

### ISSUE THREAD

**Issue by robertgarrigos** [OpenKratia/openkratia#2]
I want to work on these things before I push a first workable commit, so anyone wishing to participate or try it has something more than words:

- [ ] Navigation proof of concept
- [ ] Two spaces with minimum functionality: processes and assemblies
- [ ] Two components with minimum functionality: Debates and polls
- [ ] Components sorting by rate
- [ ] Pack all this on a profile with some example content

If you want to know what spaces and components are, look at the [structure of decidim](https://docs.decidim.org/en/features/general-description/) which I will follow essentially.

**Comment by robertgarrigos** [comment]
I pushed some code with this navigation proof of concept. If you want to try it out: install it just like you would do with [backdrop cms](https://backdropcms.org/installation). At the of installation, you will be able to setup the profile: chose 'public' and 'devel'. That's it.

### What you'll find it

For now, just a way to navigate through the different spaces and components. The spaces and components have not much functionality yet, they are just basic backdrop node types. This will come in the next iteration. The Assembly space you will see is not yet fully navigable as the Process space. So, use the process to see how to navigate through all its components, nodes and edit forms.

Also, if you look at the code, you will find plenty of commented out code and other not used code, due to the test of the navigation testing I've been doing.

I've tried to have a simple navigation as possible, based on tabs (which have a limitation of two levels) and with some context info top show the user where she is at any time when browsing a node, whether it is a space, a component or a normal backdrop node not related with any space or component. All this information is presented in the whole website based on a level system: 
1. Header with spaces and other menu items in the main navigation menu.
2. Space context info, if needed.
3. Space actions (components access and config), both as normal user and admin, with a tabbed menu.

<img width="630" alt="Your_first_process____OpenKratia" src="https://user-images.githubusercontent.com/105581/89706872-dcf41a80-d969-11ea-8dd7-8538b0a49a70.png">

Feel free to let me know what are your thoughts on this, try it out and file any issue about it. Thanks.

**Comment by stpaultim** [comment]
FYI - I was able to download and install this early version of OpenKratia on my laptop, using Lando. No problems during installation. 

I think I'll be able to help more if we break apart some of the components/processes into separate issues where I can ask specific questions and provide comments. 

At first, I did not find this page "http://localhost:32791/?q=admin/okratia/spaces" and I wasn't sure what I was looking at. However, following menu "Open Kratia >> Spaces" was important to seeing how the site is structured (I think). This page was interested and is helpful for me to start to understand how the site works. 

I look forward to the next steps. In the meantime, I'm going to keep asking questions and open a few issues to help (hopefully) create a better understanding of where this project is going. Let me know if I am being a bother/pain. :-)

**Comment by robertgarrigos** [comment]
Yes, I understand that it might be difficult to understand how everything works. Even for me, there are many things I don't know yet how they'll work. You will find, for instance, many blank config pages, as I'm trying to get a clear navigation process, both for viewing and editing content.

An example: I updated the code just today because the initial navigation to edit a space triggered an error with ajax calls within the edit form. Thus, I left the node/%/edit menu as original and added a new config tab to access the configuration area for the space. Then, I show a yellow background for the header just as a simple way now to show whether we are in an admin area of any content or not but always show a space context block any time we are seeing an Open Kratia related content. This way it will evident when you are navigating or editing an Open Kratia content or not. Later we might see how to apply a better design.

I'll concentrate on the minimum functionality now, so it'll be easier to understand how it everything works. 

Good idea about the separated issues. I've already begun to answer some of your questions.
